spec.bs

```diff
@@ -524,7 +528,7 @@ to discern when a pending config is finalized, it is important that all visibili
 transparent fields do not change from the pending config to the finalized config, given that they
 can be inspected through {{FencedFrameConfig}}'s getters. Therefore, a {{FencedFrameConfig}} that
 is created and exposed to the web platform is effectively immutable even if its underlying
-[=fencedframeconfig/config=] is technically "pending", and will finish resolving completely later.
+[=fenced frame config=] is technically "pending", and will finish resolving completely later.
```

Comment:
Hmm, well now a fenced frame element doesn't directly have a fenced frame config struct, so maybe we should make this more specific by referencing `the [=fenced frame config=] represented by the [=fencedframe/config=]'s [=fencedframeconfig/urn=]`. WDYT?

Reply:
That works for me.